packages/cloud_firestore/cloud_firestore_web/lib/cloud_firestore_web.dart

```diff
@@ -0,0 +1,96 @@
+// Copyright 2017, the Chromium project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:cloud_firestore_platform_interface/cloud_firestore_platform_interface.dart';
+import 'package:firebase/firebase.dart' as firebase;
+import 'package:firebase/firestore.dart' show Firestore, Settings;
+import 'package:firebase_core/firebase_core.dart';
+import 'package:flutter_web_plugins/flutter_web_plugins.dart';
+
+import 'package:cloud_firestore_web/src/collection_reference_web.dart';
+import 'package:cloud_firestore_web/src/field_value_factory_web.dart';
+import 'package:cloud_firestore_web/src/document_reference_web.dart';
+import 'package:cloud_firestore_web/src/query_web.dart';
+import 'package:cloud_firestore_web/src/transaction_web.dart';
+import 'package:cloud_firestore_web/src/write_batch_web.dart';
+
+/// Web implementation for [FirestorePlatform]
+/// delegates calls to firestore web plugin
+class CloudFirestoreWeb extends FirestorePlatform {
```

Comment:
Despite the filename, I think this class should be `FirestoreWeb` rather than `CloudFirestoreWeb`.

The package was originally named `firebase_firestore` and I renamed it `cloud_firestore` in response to an internal request (see https://github.com/flutter/flutter/issues/12388). My understanding that firebase_firestore is now the preferred package name and that classes shouldn't be using the a `Cloud` prefix. At some point we might want to rename this package back (updating to 1.0 might be the right time for that). I filed https://github.com/FirebaseExtended/flutterfire/issues/1964 as a tracking issue.

Reply:
Sure. I've reverted the class name change.